lib/tutorials/en_US/getting-started.md

```diff
@@ -223,29 +94,16 @@ process.on('unhandledRejection', (err) => {
 });
 
 init();
-
 ```
 
-Now when the server is started you'll see something like:
-
-```sh
-[2017-12-03T17:15:45.114Z] INFO (10412 on box): server started
-    created: 1512321345014
-    started: 1512321345092
-    host: "localhost"
-    port: 3000
-    protocol: "http"
-    id: "box:10412:jar12y2e"
-    uri: "http://localhost:3000"
-    address: "127.0.0.1"
-```
+Save the above as `index.js` and start the server with the command `node index.js`. Now you'll find that if you visit `http://localhost:3000` in your browser, you'll see the text 'Hello, World!'.
 
-And if we visit [http://localhost:3000/](http://localhost:3000/) in the browser, we can see that logs about the request are being printed in the terminal.
+The `method` property can be any valid HTTP method, array of HTTP methods, or an asterisk to allow any method. 
 
-The behavior of the logger is configured in `options` passed to the register function.
+The `path` property defines the path including parameters. It can contain optional parameters, numbered parameters, and even wildcards. For more details, see the routing tutorial.
 
-Great! This is just one short example of what plugins are capable of, for more information check out the [plugins tutorial](/tutorials/plugins).
+The `handler` function performs the main business logic of the route and sets the response. The `handler` must return a value, a promise, or throw an error.
 
-### Everything else
+## <a name="everything" /> Everything Else
 
 hapi has many, many other capabilities and only a select few are documented in tutorials here. Please use the list to your right to check them out. Everything else is documented in the [API reference](/api) and, as always, feel free to ask questions on [github](https://github.com/hapijs/discuss/issues) or just visit us on [slack](https://join.slack.com/t/hapihour/shared_invite/enQtNTA5MDUzOTAzOTU4LTUyZmFiYjkyMTBmNDcyMmI2MmRjMzg4Y2YzNTlmNzUzNjViN2U1NmYyY2NjYjhiYWU4MGE2OTFhZDRlYWMyZDY).
```

Comment:
I would add a mention to encourage people to check the plugins tutorials so they have an understanding of how to better organize their application and add features to their server.

```suggestion
hapi has many, many other capabilities and only a select few are documented in tutorials here. This tutorial was intentionally minimal, we highly recommend you to check out the [Plugins tutorial](/tutorials/plugins). It will give some more knowledge to better organize your hapi project. We also have other tutorials of interest, please use the list to your right to check them out. Everything else is documented in the [API reference](/api) and, as always, feel free to ask questions on [github](https://github.com/hapijs/discuss/issues) or just visit us on [slack](https://join.slack.com/t/hapihour/shared_invite/enQtNTA5MDUzOTAzOTU4LTUyZmFiYjkyMTBmNDcyMmI2MmRjMzg4Y2YzNTlmNzUzNjViN2U1NmYyY2NjYjhiYWU4MGE2OTFhZDRlYWMyZDY).
```

Reply:
Fixing now